v9.2/bingo.html

```diff
@@ -118,11 +118,11 @@ <h1>About</h1>
 <footer>
```

Comment:
Let's just remove the footer entirely for now; we can add it on a style refresh, if we want.

Reply:
(There's also some quick cleanup to be done on the CSS to remove some unused values, but that could be done separately)